src/UnscentedKalmanInversion.jl

```diff
@@ -170,6 +170,31 @@ function Unscented(
     )
 end
 
+function Unscented(
+    prior::ParameterDistribution;
+    α_reg::FT = 1.0,
+    update_freq::IT = 1,
+    modified_unscented_transform::Bool = true,
+    sigma_points::String = "symmetric",
+) where {FT <: AbstractFloat, IT <: Int}
+
+    u0_mean = Vector{FT}(mean(prior)) # mean of unconstrained distribution
+    uu0_cov = Matrix{FT}(cov(prior)) # cov of unconstrained distribution
+
+    return Unscented(
+        u0_mean,
+        uu0_cov;
+        α_reg = α_reg,
+        update_freq = update_freq,
+        modified_unscented_transform = modified_unscented_transform,
+        prior_mean = u0_mean,
+        sigma_points = sigma_points,
+    )
+
+end
```

Comment:
```suggestion
function Unscented(prior::ParameterDistribution; kwargs...)
    u0_mean = Vector(mean(prior)) # mean of unconstrained distribution
    uu0_cov = Matrix(cov(prior)) # cov of unconstrained distribution
    return Unscented(u0_mean, uu0_cov; kwargs...)
end
```
A bit shorter

Reply:
This almost works, I think we will need to do 
```julia
return Unscented(u0_mean, uu0_cov; prior_mean = u0_mean, kwargs...)
```

Reply:
I'll just do your suggestion for now as there is no benefit of mine really...